docs/quickstarts/react.mdx

```diff
@@ -41,216 +92,79 @@ Below is an example of an .env file. If you are signed into your Clerk Dashboard
-</Callout>
-
-
-#### Using a routing library with Clerk
-
-Clerk can work with any routing library. Below is an example of how to use Clerk with React Router and how to embed the Clerk components in your application.
-
-```tsx filename="app.tsx"
-import {
-  ClerkProvider,
-  SignedIn,
-  SignedOut,
-  RedirectToSignIn,
-  SignIn,
-  SignUp,
-  UserButton,
-} from "@clerk/clerk-react";
-import { BrowserRouter, Route, Routes, useNavigate } from "react-router-dom";
-
-if (!process.env.REACT_APP_CLERK_PUBLISHABLE_KEY) {
-  throw new Error("Missing Publishable Key")
-}
-
-const clerkPubKey = process.env.REACT_APP_CLERK_PUBLISHABLE_KEY;
-
-function PublicPage() {
-  return (
-    <>
-      <h1>Public page</h1>
-      <a href="/protected">Go to protected page</a>
-    </>
-  );
-}
-
-function ProtectedPage() {
-  return (
-    <>
-      <h1>Protected page</h1>
-      <UserButton />
-    </>
-  );
-}
-
-function ClerkProviderWithRoutes() {
-  const navigate = useNavigate();
-
-  return (
-    <ClerkProvider
-      publishableKey={clerkPubKey}
-      navigate={(to) => navigate(to)}
-    >
-      <Routes>
-        <Route path="/" element={<PublicPage />} />
-        <Route
-          path="/sign-in/*"
-          element={<SignIn routing="path" path="/sign-in" />}
-        />
-        <Route
-          path="/sign-up/*"
-          element={<SignUp routing="path" path="/sign-up" />}
-        />
-        <Route
-          path="/protected"
-          element={
-          <>
-            <SignedIn>
-              <ProtectedPage />
-            </SignedIn>
-            <SignedOut>
-              <RedirectToSignIn />
-            </SignedOut>
-          </>
-          }
-        />
-      </Routes>
-    </ClerkProvider>
-  );
-}
-
-function App() {
-  return (
-    <BrowserRouter>
-      <ClerkProviderWithRoutes />
-    </BrowserRouter>
-  );
+      <SignedIn>
+        <UserButton afterSignOutUrl="/" />
+      </SignedIn>
+    </div>
+  )
 }
 
-export default App;
+export default App
 ```
 
-### Read session and user data
-
-Clerk provides a set of [hooks and helpers](/docs/references/react/use-user) that you can use to access the active session and user data in your React application. Here are examples of how to use these helpers.
-
-#### `useAuth`
-
-The [`useAuth`](/docs/references/react/use-auth) hook is a convenient way to access the current auth state. This hook provides the minimal information needed for data-loading and helper methods to manage the current active session.
-
-```tsx filename="pages/example.tsx"
-import { useAuth } from "@clerk/clerk-react";
-
-export default function Example() {
-  const { isLoaded, userId, sessionId, getToken } = useAuth();
-
-  // In case the user signs out while on the page.
-  if (!isLoaded || !userId) {
-    return null;
-  }
+If you removed the Vite boilerplate from `App.tsx` by copying and pasting the boce, you probably want to also delete `src/App.css` and `src/assets/react.svg`.
```

Comment:
Not sure if this is even necessary in the quickstart - the users can decide how they want to clean up/maintain their app. This isn't something that affects how Clerk gets up and running for this tutorial.

Reply:
I think its worth the mention. I'm fine with removing it though.